android/source/VideoLocker/src/org/edx/mobile/base/BaseFragmentActivity.java

```diff
@@ -638,4 +638,9 @@ protected void showWebDialog(String fileName, boolean showTitle, String dialogTi
             webViewFragment.show(getSupportFragmentManager(), "dialog");
         }
     }
+
+    protected void hideSoftKeypad() {
+        getWindow().setSoftInputMode(
```

Comment:
For the screen which have input views like <code>EditText</code>, the soft keypad appears automatically as the first input view on the screen gets focus.
We have to keep the keypad hidden at the launch of screen and then user an tap on an input view and keypad appears.

So, this lets the keypad be hidden by default.